code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -51,6 +51,13 @@ var/list/hardcoded_gases = list("o2","n2","co2","plasma") //the main four gases,
 	gas_overlay = "plasma"
 	moles_visible = MOLES_PLASMA_VISIBLE
 
+/datum/gas/freon
+	id = "freon"
+	specific_heat = 1
```

Comment:
correct, I don't think you understand how specific_heat works. This isn't a temperature, but rather how well it stores temperature or lack of temperature.

With your current values here, 1 canister of o2 at room temperature could heat up 20 canisters of freon.

Set this to higher for freon, 2389 to be exact.